packages/graphql-toolbox/src/modules/HelpDrawer/HelpDrawer.tsx

```diff
@@ -199,10 +66,10 @@ export const HelpDrawer = ({ onClickClose, schema }: Props) => {
                         >
                             {"\u2715"}
                         </span>
-                    </Fragment>
+                    </>
                 </div>
             ) : null}
-            <Fragment>
+            <>
```

Comment:
You should be able to remove this fragment as there is a div parent.

Reply:
430650b1e